clients/client-java/src/main/java/org/apache/gravitino/client/GenericTopic.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.client;
+
+import com.google.common.base.Joiner;
+import java.util.Map;
+import org.apache.gravitino.Audit;
+import org.apache.gravitino.MetadataObject;
+import org.apache.gravitino.MetadataObjects;
+import org.apache.gravitino.Namespace;
+import org.apache.gravitino.dto.messaging.TopicDTO;
+import org.apache.gravitino.messaging.Topic;
+import org.apache.gravitino.tag.SupportsTags;
+
+/** Represents a generic topic. */
+public class GenericTopic implements Topic, SupportsTagOperations {
```

Comment:
So what does the word `Generic` mean here? Just as it has implemented interface `SupportsTagOperations`?

Reply:
Yeah, the purpose is to implement the `SupportsTagOperations`.